src/containers/floating/Settings/sections/general/LocalNodeSync.tsx

```
@@ -0,0 +1,122 @@
+    p {
+        color: ${({ theme }) => theme.palette.text.secondary};
+    }
+`;
+
+const ProgressWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+`;
+
+interface Progress {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Progress'. Consider to delete it or rename it.

'Progress' is defined here:


(lint/suspicious/noRedeclare)